### ISSUE TITLE
Crash when pressing button in presented view with GeometryReader and TabView

### ISSUE THREAD

**Issue by otymartin** [nathantannar4/Transmission#61]
@nathantannar4 

Pressing the "Dismiss" button in the Presented View causes the following crash:
`SwiftUICore/GestureContainerFactory.swift:25: Fatal error: Gesture container factory must be configured`

```swift
import SwiftUI
import Transmission

struct ContentView: View {
    @State var show = false
    var body: some View {
        GeometryReader { context in
            TabView {
                VStack {
                    Button {
                        withAnimation {
                            show.toggle()
                        }
                    } label: {
                        Text("Show Slide")
                    }
                }
                .padding()
            }
            .tabViewStyle(.page)
            .onTapGesture {
                show.toggle()
            }
            .presentation(transition: .slide(edge: .bottom), isPresented: $show) {
                    Button { } label: { 👈 Crash when pressed.
                        Text("Dismiss")
                            .foregroundStyle(.white)
                    }
            }

        }
    }
}
```

**Comment by otymartin** [comment]
What appears to resolve the issue is placing the `.presentation(....)` modifier on the GeometryReader instead eg
```swift
GeometryReader {
  ...
}
.presentation(transition: .slide(edge: .bottom), isPresented: $show) {
    Button {
    } label: {
        Text("Dismiss")
            .foregroundStyle(.white)
    }
}

```

**Comment by nathantannar4** [comment]
Fixed in `1.4.10`
